dev/app-shell/elements/persistent-arc.html

```diff
@@ -16,16 +16,32 @@
         db: db.child('arcs')
       };
     }
+    _getAdditionalRecipe() {
+      const params = (new URL(document.location)).searchParams;
```

Comment:
Walter and I talked a bit about this.. Instead of using document.location here, we could get this via the config if we had an onConfig event wired up. We thought that might require bubbling the events down from arc-app, or using props/state, but that path didn't seem clear and required more work than this. @sjmiles we thought you might have more thoughts, though.